applications/StructuralMechanicsApplication/custom_constitutive/constitutive_laws_integrators/generic_constitutive_law_integrator_damage.h

```diff
@@ -145,6 +145,7 @@ class GenericConstitutiveLawIntegratorDamage
             KRATOS_ERROR << "SOFTENING_TYPE not defined or wrong..." << softening_type << std::endl;
             break;
         }
+        rDamage = (rDamage > 0.999) ? 0.999 : rDamage;
```

Comment:
Do you think that it makes sense to define a variable to define the thereshold in the properties?

Reply:
I think that it can be done but almost everybody should be fine with 0.999 In think

Reply:
I've solved this because @SergioJimenezReyes had some trouble without this condition (I forgot to add it)